lib/omnibus/packagers/msi.rb

```diff
@@ -407,6 +410,17 @@ def wix_candle_extensions
     end
 
     #
+    # Returns the options to use for candle
+    #
+    # @return [Array]
+    #   the extensions that will be loaded for candle
+    #
+    def wix_candle_flags
+      # we support x86 or x64.  No Itanium support (ia64).
+      @wix_candle_flags ||= "-arch " + (Config.windows_arch == :x86 ? "x86" : "x64")
```

Comment:
You can use "to_s" to convert the symbol to a string, but the fact that the symbol names correspond to the candle flags is a coincidence...  so *shrug*.  You can leave it as is if you feel it's cleaner.